packages/taquito/src/operations/operation-errors.ts

```diff
@@ -125,8 +126,13 @@ export const flattenErrors = (
     for (let j = 0; j < results[i].contents.length; j++) {
```

Comment:
nitpick, kinda feel like refactor the entire flattenErrors function, don't like the nested if and for loops 😅

Reply:
I agree, I also think that it can probably be written better. But since it's non-blocking and it's logically still correct, I would rather leave it be. What do you think?

Reply:
I understand, let's leave it for now